packages/terra-application-utility/README.md

```diff
@@ -4,7 +4,7 @@
 [![NPM version](http://img.shields.io/npm/v/terra-application-utility.svg)](https://www.npmjs.org/package/terra-application-utility)
 [![Build Status](https://travis-ci.org/cerner/terra-framework.svg?branch=master)](https://travis-ci.org/cerner/terra-framework)
 
-{insert description}
```

Comment:
Could we add a more detailed description on the use and purpose of this component? There are two types defined and its a bit unclear how these correlate to the header and menu layouts referenced. We should specify directly these  are the terra-appication-header-layout, etc.

Reply:
Also, try to leave the implementation details out of it. I don't think you need to mention anything about building a configuration. That can be left to the documentation details.